infrastructure/cli/BUILD

```diff
@@ -26,3 +53,24 @@ go_test(
     embed = [":cli_lib"],
     deps = ["//infrastructure/cli/pkg/tests"],
 )
+
+genrule(
```

Comment:
Are you sure that these rules are necessary given that they just cp 1:1? 🤔

Reply:
Yes to make the path independent from the os it is build on

Reply:
Ok, so up to you, but you can also use `$(bazel info bazel-genfiles)` to obtain the path to the out folder without having to hardcode the os specific path

Reply:
Thanks, Chris I will keep it in mind 👍